src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -198,6 +191,21 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     },
   };
 
+  @computed get isNamespaced(): boolean {
+    const { store, dependentStores = [] } = this.props;
```

Comment:
I think we should not check `dependentStores` here (noticed this when you opened #2950).

Reply:
Okay will change this then and close #2950. I will leave the other bug to be fixed by #2755.